src/Sulu/Bundle/ContactBundle/Api/Account.php

```diff
@@ -1066,9 +1066,24 @@ public function getLogo()
      *
      * @param MediaInterface $media
      *
+     * @deprecated Please use setMedia instead.
+     *
      * @return Account
+     *
      */
     public function addMedia(MediaInterface $media)
+    {
+        return $this->setMedia($media);
+    }
+
+    /**
+     * Set media.
+     *
+     * @param MediaInterface $media
+     *
+     * @return Account
+     */
+    public function setMedia(MediaInterface $media)
```

Comment:
As @alexander-schranz already mentioned, this is not doing what I would expect it to do. I would remove the above code again, since this is just about renaming the `getMedias` method.